samples/sample_common/include/sample_defs.h

```diff
@@ -45,6 +45,27 @@ enum {MCTF_BITRATE_MULTIPLIER = 100000};
 #endif
 #endif
 
+
+#if defined(WIN32) || defined(WIN64)
+
+enum {
+    MFX_HANDLE_DEVICEWINDOW  = 0x101 /* A handle to the render window */
+}; //mfxHandleType
+
+#ifndef D3D_SURFACES_SUPPORT
+#define D3D_SURFACES_SUPPORT 1
+#endif
+
+#if defined(_WIN32) && !defined(MFX_D3D11_SUPPORT)
```

Comment:
1. Do we use WIN32 or _WIN32? This is inconsistent with line 49.
2. Probably MFX_D3D11_SUPPORT should be enough by itself here? We are already covered by WIN32/64 above and we define D3D11_SUPPORT only for windows...

Reply:
This is synchronization patch and it will include windows parts as well. After full migration of samples to open source windows-related parts of the code will be uploaded into GitHub. So, no need to remove them now